benchmark/utils/cuda_linops.hpp

```diff
@@ -710,10 +709,10 @@ using cusp_gcoo = detail::CuspGenericCoo<etype>;
 
 #if defined(CUDA_VERSION) && (CUDA_VERSION < 11000)
 using cusp_coo =
-    detail::CuspHybrid<etype, gko::int32, CUSPARSE_HYB_PARTITION_USER, 0>;
+    detail::CuspHybrid<etype, itype, CUSPARSE_HYB_PARTITION_USER, 0>;
```

Comment:
do they support int64?

Reply:
I am not certain, I mainly wanted to avoid compilation issues with the wrong matrix_data index type if we do add int64 benchmarks at some point.

Reply:
I see, when we really this for int64 and get error, we can add the throw at that point